### PR TITLE
lots of ui changes for a serverless future

### LINKED ISSUE #5255: Update homepage
The homepage should reflect the new status of the server.

Note the onboarding tour still goes to the server homepage, and that should stay working until we've updating Firefox.

### LINKED ISSUE #5257: Remove upload references from onboarding
The onboarding should not refer to Save or Uploading

### LINKED ISSUE #5258: Review download/copy buttons
Without a Save button, we should consider if we want to make any changes to the download and copy buttons, or add a label to one of them.

### LINKED ISSUE #5300: Review download-only-mode
Since everyone is in download only mode soon, we should review the code and make sure we don't have anything exceptional for those users, including any messaging specific to that mode.

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #5257 
Partially Fixes #5300 
Fixes #5258 
Fixes #5255

### CODE REVIEW

File: locales/en-US/webextension.properties [hunk 1]
```diff
@@ -58,7 +58,7 @@ genericErrorTitle = Whoa! Firefox Screenshots went haywire.
 genericErrorDetails = We’re not sure what just happened. Care to try again or take a shot of a different page?
 # Section for onboarding strings
 # Slide number 1:
-tourBodyIntro = Take, save, and share screenshots without leaving Firefox.
+tourBodyIntroServerless = Take screenshots without leaving Firefox.
```
Comment (johngruen) [inline locales/en-US/webextension.properties]: @ianb michelle wanted to review this string before we merge

Comment (6a68) [reply locales/en-US/webextension.properties]: @johngruen Probably need that review today, or tomorrow at latest, to land this in mozilla-central before the end of week deadline

**Comment by ianb** [comment]
Tests are fixed. We'll want to get @flodolo's review, and @johngruen should confirm the string changes

File: locales/en-US/webextension.properties (hunk 1 above)
Comment (ianb) [reply locales/en-US/webextension.properties]: The new string is per feedback from Meridel (who is working on this with us instead of Michelle)

**Comment by punamdahiya** [comment]
On non-firefox browser shot page, upsell string is replaced by text gScreenshotsDescription⁩

screenshot 2019-01-17 10 17 38

**Comment by punamdahiya** [comment]
Other than issue called out in [above comment](https://github.com/mozilla-services/screenshots/pull/5301#issuecomment-455274682),  serverless UI changes looks good. It will be good to have a solid QA for download only mode. Thanks!

**Pull request merged**